### DISCUSSION TITLE
snippets search function

### DISCUSSION THREAD

**Discussion by AIcyberark** [isontheline/pro.webssh.net#720] [Ideas]
I need snippets search function, this is really useful.It would be better if webssh.pro added batch execution server commands.These are the advantages supported by servercat

**Comment by isontheline** [comment]
Hello @AIcyberark 👋

Thank you a lot for your feedback!

Batch execution is planned here : https://github.com/isontheline/pro.webssh.net/issues/564

I have created a new issue for the ability yo search Snippets : https://github.com/isontheline/pro.webssh.net/issues/721
